src/services/transaction-apis.js

```diff
@@ -1,11 +1,4 @@
-import { TRANSACTION_ID_PLACEHOLDER } from '../constants';
-import { TRANSACTIONS_APIS_URLS } from '../constants/api';
-
-export function buildTransactionApiUrl (apiName, transactionId, testApi = false) {
-  const api = TRANSACTIONS_APIS_URLS[apiName];
-  if (!api) {
-    throw new Error(`API ${apiName} is not listed`);
-  }
-  const apiUrl = testApi ? api.testnet : api.mainnet;
-  return apiUrl.replace(TRANSACTION_ID_PLACEHOLDER, transactionId);
+export function buildTransactionServiceUrl ({ serviceUrls, searchValue = '', newValue = '', testApi = false }) {
```

Comment:
I think for the sake of better understanding the `searchValue` and `newValue` should be renamed to something more explicit.

I think `searchValue` could remain something local to this method since this is the same always. That way this method does not have to know: https://github.com/blockchain-certificates/cert-verifier-js/pull/493/files#diff-ad67ee00dc3bac28ca5ebde6ee4f8e69R12. 
Unless you are paving the way for something different? In which case I would rename `searchValue` to `transactionIdPlaceholder` since this is always what we want to replace.

Similarly, I'd say `newValue` needs to be called `transactionId`, since this is what it is.

Reply:
I was thinking about making the function agnostic so it can be used with any public services we use. For example, Etherscan has an `action` parameter, that uses [3 different values](https://github.com/blockchain-certificates/cert-verifier-js/blob/master/src/explorers/ethereum.js#L9).
This would allow the app to generate URLs from any template, but since we pick the API URL from `main` or `test`, we'll have to either get that out, or update the signature to receive an object of parameters (keys & values to replace) instead of a single value.

Reply:
I'll rename the parameters for now until the core logic of `buildTransactionServiceUrl` is changed.